test/powershell/Language/Interop/DotNet/DotNetAPI.Tests.ps1

```diff
@@ -6,8 +6,8 @@ Describe "DotNetAPI" -Tags "CI" {
     $posh_pi = 3.14159265358979
 
     It "Should be able to use static .NET classes and get a constant" {
-	[System.Math]::E  | Should -Match $posh_E.ToString()
-	[System.Math]::PI | Should -Match $posh_pi.ToString()
+	[System.Math]::E  | Should -Match $posh_E.ToString("G15")
```

Comment:
The semantics in this test seem hard to reason about; it expects a numerical constant, stringified by language primitives to be equal to a particular direct stringification when it's turned into a regex.

That seems like a good test (except for the regex bit), but doesn't feel like what the name of the test implies, which I would write like:

```powershell
It "Should be able to use static .NET classes and get a constant" {
    [System.Math]::E | Should -Be 2.7182818284590451
    [System.Math]::PI | Should -Be 3.1415926535897931
}
```

Reply:
And then it's useful to keep the `ToString(...)` tests as a different .NET API test

Reply:
Sure, will do that.